pages/lessons.md

```diff
@@ -35,11 +35,9 @@ Data Carpentry workshops are domain-specific, so that we are teaching researcher
 
 ## Community-Contributed materials
 
-These are Carpentries style lessons that have been contributed by community members and are not part of Data Carpentry's official offerings.
-
-- [Python for Atmosphere and Ocean Scientists](#python-atmos)
-
-
+<p>
+The Carpentries also shares <a href="https://carpentries.org/community-lessons/">The Carpentries Community Developed Lessons</a>.  This includes <a href="https://carpentries.org/community-lessons/#the-carpentries-incubator">The Carpentries Incubator</a> (lessons under development and seeking peer review), and <a href="https://carpentries.org/community-lessons/#the-carpentrieslab">The CarpentriesLab</a> (lessons that have been approved by The Carpentries but are not part of our standard offerings).   
```

Comment:
Please change "have been approved" to "have been vetted" and then go ahead and merge. I'll make the corresponding change to the LC website since I already merged that PR